compiler/hash-utils/Cargo.toml

```diff
@@ -24,10 +24,21 @@ itertools = "0.10.5"
 lazy_static = "1.4.0"
 libc = "0.2"
 log = "0.4"
-num-traits = "0.2"
 num-bigint = "0.4"
+num-traits = "0.2"
 parking_lot = "0.12"
 rayon = "1.5.0"
 smallvec = "1.11.0"
 stacker = "0.1.15"
 thin-vec = "0.2.10"
+
+[target.'cfg(windows)'.dependencies.windows]
+version = "0.52.0"
+features = [
+    "Win32_Foundation",
+    "Win32_Storage_FileSystem",
+    "Win32_System_IO",
+    "Win32_System_ProcessStatus",
+    "Win32_System_Threading",
```

Comment:
wtf

Reply:
Yeh its like Microsoft doing it their way